### PR TITLE
Added chaos jobs for Elasticsearch  and Updated litmus script path in deploy-openebs

### PR DESCRIPTION
**Pull request by sushma1118**
* Added chaos job cstor-pool-delete
* Added chaos job pool-network-loss
* Updated litmus script path in deploy-openebs

@gprasath @nsathyaseelan

### CODE REVIEW

File: Openshift-EE/pipelines/ElasticSearch/stages/4-chaos/IOWW-cstor-pool-delete/cstor-pool-delete [hunk 1]
```diff
@@ -1,5 +1,101 @@
...
+time="date"
+current_time=$(eval $time)
+
+present_dir=$(pwd)
+echo $present_dir
+#Creating e2e custom resource result for the test.
+bash Openshift-EE/utils/e2e-cr jobname:pool-delete jobphase:Waiting 
+bash Openshift-EE/utils/e2e-cr jobname:pool-delete jobphase:Running init_time:"$current_time" jobid:"$job_id" pipelineid:"$pipeline_id" testcaseid:"$case_id"
+bash Openshift-EE/utils/e2e-cr jobname:pool-kill jobphase:Waiting      #cstor-pool-kill job should not be executed till pool-delete is completed
+
```
Comment (nsathyaseelan) [inline Openshift-EE/pipelines/ElasticSearch/stages/4-chaos/IOWW-cstor-pool-delete/cstor-pool-delete]: @sushma1118 create cr status as waiting for `jobname:app-pod-failure jobphase:Waiting`

File: Openshift-EE/pipelines/ElasticSearch/stages/4-chaos/638K-cstor-pool-kill/cstor-pool-kill [hunk 2]
```diff
@@ -1,5 +1,86 @@
...
+time="date"
+current_time=$(eval $time)
+
+present_dir=$(pwd)
+echo $present_dir
+
+#Creating e2e custom resource result for the test.
+
+bash Openshift-EE/utils/pooling jobname:pool-delete    #wait till completion of cstor-pool-delete
+bash Openshift-EE/utils/e2e-cr jobname:pool-kill jobphase:Waiting
```
Comment (nsathyaseelan) [inline Openshift-EE/pipelines/ElasticSearch/stages/4-chaos/638K-cstor-pool-kill/cstor-pool-kill]: @sushma1118 remove the step creating cr for pool-kill

File: Openshift-EE/pipelines/ElasticSearch/stages/4-chaos/638K-cstor-pool-kill/cstor-pool-kill [hunk 3]
```diff
@@ -1,5 +1,86 @@
...
+
+present_dir=$(pwd)
+echo $present_dir
+
+#Creating e2e custom resource result for the test.
+
+bash Openshift-EE/utils/pooling jobname:pool-delete    #wait till completion of cstor-pool-delete
+bash Openshift-EE/utils/e2e-cr jobname:pool-kill jobphase:Waiting 
+bash Openshift-EE/utils/e2e-cr jobname:pool-kill jobphase:Running init_time:"$current_time" jobid:"$job_id" pipelineid:"$pipeline_id" testcaseid:"$case_id"
+bash Openshift-EE/utils/e2e-cr jobname:app-pod-failure jobphase:Waiting #don't execute app-pod-failure till this job is completed
```
Comment (nsathyaseelan) [inline Openshift-EE/pipelines/ElasticSearch/stages/4-chaos/638K-cstor-pool-kill/cstor-pool-kill]: @sushma1118 remove the step creating cr for app-pod-failure and include this step in pool-delete script

File: Openshift-EE/pipelines/ElasticSearch/stages/4-chaos/401P-elasticsearch-pod-failure/elasticsearch-pod-failure [hunk 4]
```diff
@@ -1,5 +1,99 @@
...
+
+time="date"
+current_time=$(eval $time)
+
+present_dir=$(pwd)
+echo $present_dir
+
+#Creating e2e custom resource result for the test.
+bash Openshift-EE/utils/pooling jobname:pool-kill     # wait till cstor-pool-kill is completed 
+bash Openshift-EE/utils/e2e-cr jobname:app-pod-failure jobphase:Waiting
```
Comment (nsathyaseelan) [inline Openshift-EE/pipelines/ElasticSearch/stages/4-chaos/401P-elasticsearch-pod-failure/elasticsearch-pod-failure]: @sushma1118 remove the step creating cr for app-pod-failure.

File: Openshift-EE/pipelines/ElasticSearch/stages/4-chaos/IOWW-cstor-pool-delete/cstor-pool-delete (hunk 1 above)
Comment (sushma1118) [reply Openshift-EE/pipelines/ElasticSearch/stages/4-chaos/IOWW-cstor-pool-delete/cstor-pool-delete]: ok

File: Openshift-EE/pipelines/ElasticSearch/stages/4-chaos/638K-cstor-pool-kill/cstor-pool-kill (hunk 2 above)
Comment (sushma1118) [reply Openshift-EE/pipelines/ElasticSearch/stages/4-chaos/638K-cstor-pool-kill/cstor-pool-kill]: OK

File: Openshift-EE/pipelines/ElasticSearch/stages/4-chaos/638K-cstor-pool-kill/cstor-pool-kill (hunk 3 above)
Comment (sushma1118) [reply Openshift-EE/pipelines/ElasticSearch/stages/4-chaos/638K-cstor-pool-kill/cstor-pool-kill]: OK

File: Openshift-EE/pipelines/ElasticSearch/stages/4-chaos/401P-elasticsearch-pod-failure/elasticsearch-pod-failure (hunk 4 above)
Comment (sushma1118) [reply Openshift-EE/pipelines/ElasticSearch/stages/4-chaos/401P-elasticsearch-pod-failure/elasticsearch-pod-failure]: ok

**Pull request closed without merging**